WebView2: Add documentation for .NET support (#627)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/releasenotes.md

> title: Microsoft Edge WebView 2 for Win32 apps Release Notes

Reviewer: ```suggestion
title: Release Notes for Microsoft Edge WebView2 for Win32, WPF, and WinForms
```

Release notes for [WebView2 SDK][WebView2NuGetGallery].  

## 0.X.XXX

> [NuGet package][WebView2NuGetGallery0.X.XXX] | minimum Microsoft Edge version 8X.0.XXX.X.

Reviewer: Missing version numbers?

Author: Added in a later commit.

> **Re-compile your app after updating the NuGet package.** 

Reviewer: ```suggestion
Re-compile your app after updating the NuGet package.
```

> *   **Announcement:** WebView2 now supports WinForms and WPF on .NET Framework 4.6.2 or later and .NET Core 3.0 or later in the **pre-release package**

Reviewer: ```suggestion
*   Announcement: WebView2 now supports WinForms and WPF on .NET Framework 4.6.2 or later and .NET Core 3.0 or later in the pre-release package.
```

Reviewer 2: explaining it is in the pre-release package is really important and we want developers to know this - this is very unqiue to their development experience

Reviewer: For emphasis, the MS Style Guide says to use italics.

*   Checkout the [WPF Getting Started Guide]() to get started building WPF applications and our [WPF Reference]() for WPF specific APIs
*   Checkout the [WinForms Getting Started Guide]() to get started building WinForms applications and our [WinForms Reference]() for WinForms specific APIs
*   Checkout [.NET Reference]() for CoreWebView2 APIs 

> *   **Known Issues:** We are aware of some issues in this pre-release that we will resolve in future releases

Reviewer: ```suggestion
*   Known Issues: The following is a list of known issues in the WebView2 SDK (Preview).
```

    - **DPI Awareness:** WebView2 for WPF is currently not DPI aware. When initializing WebView2 on high DPI monitors, there is a known issue where the WebView at first initializes as a fraction of the window until the WebView is resized.

>     - **WPF Designer:** This release does not currently support the WPF designer. Place the WebView2 control in your app by directly modifying the appropriate XAML in a text editor

Reviewer: ```suggestion
    - DPI Awareness: WebView2 for WPF is currently not DPI aware. When initializing WebView2 on high DPI monitors, there is a known issue where the WebView initializes as a fraction of the window until the WebView is resized.
    - WPF Designer: This release does not currently support the WPF designer. Place the WebView2 control in your app by directly modifying the appropriate XAML in a text editor
```

Reviewer 2: Hey Reeza we have traditionally bolded these titles so users skimming through the release notes have the ability to see what issues are. Not bolding these specifically would make the entire doc look inconsistent

Reviewer 2: your first bullet point also incorrectly changes the meaning of the the issue

Reviewer 2: Appreciate the comments, but will revert back to the original text
